labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,158 @@
+class CharacterCreate {
```

Comment:
Just `class Character`, please. Classes are named for what they be, not what they do. That way, when you write `new Character(...)` it reads like you're making a new, um, character.